server/api/app.py

```diff
@@ -67,19 +67,7 @@ def get_assessment_route():
     except Exception as e:
         return jsonify({"success": False, "error": str(e)}), 500
 
-
-# @app.route("/assessment")
```

Comment:
@avatyani what is the reason for removing this?

Reply:
these were old code which were commented out long time ago. I was basically cleaning the code